pkgs/io_file/src/constants.g.h

```diff
@@ -0,0 +1,95 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+// Regenerate with `dart run tool/build_constants.dart`.
+
+#include <stdint.h>
+
+// A sentinal indicating that a constant is not defined on the current platform.
+#define libc_shim_UNDEFINED 9223372036854775807
```

Comment:
Add some documentation why this value is unlikely to collide with any other defines. (Possibly use the hex representation to show it's not an obvious hex encoding.)

Reply:
Done and added asserts to the generated C code to verify that none of the constants do collide with this value.